src/test/java/uk/nhs/hee/tis/trainee/forms/service/LtftServiceTest.java

```diff
@@ -250,8 +291,443 @@ void shouldGetFilteredLocalOfficeLtftWhenFiltersNotEmpty() {
+            ))
+            .build())
+        .build();
+    entity.setContent(content);
+
+    when(ltftRepository
+        .findByIdAndStatus_Current_StateNotInAndContent_ProgrammeMembership_DesignatedBodyCodeIn(
+            any(), any(), any())).thenReturn(Optional.of(entity));
+
+    Optional<LtftFormDto> optionalDto = service.getAdminLtftDetail(ID);
+
+    assertThat("Unexpected dto presence.", optionalDto.isPresent(), is(true));
+
+    LtftFormDto dto = optionalDto.get();
+    DiscussionsDto discussions = dto.discussions();
+    assertThat("Unexpected TPD name.", discussions.tpdName(), is("Tee Pee-Dee"));
+    assertThat("Unexpected TPD email.", discussions.tpdEmail(), is("t.pd@example.com"));
+    assertThat("Unexpected other discussion count.", discussions.other(), hasSize(2));
+
+    PersonDto discussion1 = discussions.other().get(0);
+    assertThat("Unexpected discussion name.", discussion1.name(), is("Other 1"));
+    assertThat("Unexpected discussion email.", discussion1.email(), is("other.1@example.com"));
+    assertThat("Unexpected discussion role.", discussion1.role(), is("Role 1"));
+
+    PersonDto discussion2 = discussions.other().get(1);
+    assertThat("Unexpected discussion name.", discussion2.name(), is("Other 2"));
+    assertThat("Unexpected discussion email.", discussion2.email(), is("other.2@example.com"));
+    assertThat("Unexpected discussion role.", discussion2.role(), is("Role 2"));
+  }
+
+  @Test
+  void shouldGetAdminLtftChangeDetailWhenFormFound() {
+    LtftForm entity = new LtftForm();
+    entity.setId(ID);
+
+    UUID changeId = UUID.randomUUID();
+    UUID calculationId = UUID.randomUUID();
+
+    LtftContent content = LtftContent.builder()
+        .change(CctChange.builder()
+            .id(changeId)
+            .calculationId(calculationId)
+            .type(CctChangeType.LTFT)
+            .wte(0.5)
+            .startDate(LocalDate.MIN)
+            .endDate(LocalDate.EPOCH)
+            .build())
+        .build();
+    entity.setContent(content);
+
+    when(ltftRepository
+        .findByIdAndStatus_Current_StateNotInAndContent_ProgrammeMembership_DesignatedBodyCodeIn(
+            any(), any(), any())).thenReturn(Optional.of(entity));
+
+    Optional<LtftFormDto> optionalDto = service.getAdminLtftDetail(ID);
+
+    assertThat("Unexpected dto presence.", optionalDto.isPresent(), is(true));
+
+    LtftFormDto dto = optionalDto.get();
+    CctChangeDto change = dto.change();
+    assertThat("Unexpected ID.", change.id(), is(changeId));
+    assertThat("Unexpected calculation ID.", change.calculationId(), is(calculationId));
+    assertThat("Unexpected type.", change.type(), is(CctChangeType.LTFT));
+    assertThat("Unexpected WTE.", change.wte(), is(0.5));
+    assertThat("Unexpected start date.", change.startDate(), is(LocalDate.MIN));
+    assertThat("Unexpected end date.", change.endDate(), is(LocalDate.EPOCH));
+    assertThat("Unexpected CCT date.", change.cctDate(), nullValue());
```

Comment:
Is a null CCT date (one of) the thing(s) we need to refactor (as per John's comment in stand-up)?

Reply:
Yes, it's in a few places though so I'll submit it separately rather than with this PR.